src/System.Reflection.Emit/ref/System.Reflection.Emit.csproj

```diff
@@ -9,14 +9,17 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">
     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
+    <ProjectReference Include="..\..\System.Runtime.InteropServices\ref\System.Runtime.InteropServices.csproj" />
     <ProjectReference Include="..\..\System.Reflection.Primitives\ref\System.Reflection.Primitives.csproj" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'uap10.0.16299'">
     <Reference Include="netstandard" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.InteropServices" />
```

Comment:
The UWP NETNative x86 Release Build was failing because of it. We introduced a signature using the CallingConvention (no 's') enum which is defined in that contract.

Reply:
I think that's no longer the case since the API won't apply to UAP10.0.16299 now that its been moved.

Reply:
The new signature is not included for the `uap10.0.16299` TargetGroup with your recent change.